src/qml/modules/Shotcut/Controls/qmldir

```diff
@@ -4,6 +4,7 @@ ColorPicker 1.0 ColorPicker.qml
 DoubleSpinBox 1.0 DoubleSpinBox.qml
 Gauge 1.0 Gauge.qml
 GradientControl 1.0 GradientControl.qml
+GradientMapControl 1.0 GradientMapControl.qml
```

Comment:
`GradientMapControl` should not be separate from the `ui.qml` because it will never be reused. These are only for reusable components.